<ui version="4.0" >
 <class>InformerMessageEditorDialog</class>
 <widget class="QDialog" name="InformerMessageEditorDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ChangeMe!</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <property name="margin" >
    <number>4</number>
   </property>
   <property name="spacing" >
    <number>4</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="lblSubject" >
     <property name="text" >
      <string>&amp;Тема</string>
     </property>
     <property name="buddy" >
      <cstring>edtSubject</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="edtSubject" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="lblText" >
     <property name="text" >
      <string>&amp;Сообщение</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="buddy" >
      <cstring>edtText</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QTextEdit" name="edtText" />
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edtSubject</tabstop>
  <tabstop>edtText</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>InformerMessageEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>194</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel" >
     <x>156</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>InformerMessageEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel" >
     <x>311</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
